kitten_options_utils.py

```diff
@@ -1,13 +1,13 @@
 from typing import Any, Callable, Iterable, Sequence, Tuple
 
-from kitty.conf.utils import KittensKeyDefinition, key_func, parse_kittens_key
+from kitty.conf.utils import KittensKeyDefinition, KeyFuncWrapper, parse_kittens_key
```

Comment:
This will not work in the currently documented minimum compatible version of Kitty. It seems to me we could adapt the older API to the newer, maybe like this:

```
try:
    from kitty.conf.utils import KeyFuncWrapper
    func_with_args = KeyFuncWrapper[FuncArgsType]()
except ImportError:
    from kitty.conf.utils import key_func
    func_with_args, args_funcs = key_func()
    func_with_args.args_funcs = args_funcs
```

(please test if it works with both the newer and the older Kitty versions).

Alternatively, update the README.md to state the new minimum compatible version, but I prefer to reserve the requirement bump to more invasive API changes.